bot/exts/utilities/bookmark.py

```diff
@@ -95,30 +137,40 @@ async def bookmark(
         *,
         title: str = "Bookmark"
     ) -> None:
-        """Send the author a link to `target_message` via DMs."""
+        """
+        Send the author a link to the specified bookmark via DMs.
```

Comment:
I think that bookmark refers to a "pin" or a reference in the user's DMs to a target message. i.e.: the bookmark is the DM itself, which is *just* getting created in here.
```suggestion
        Send the author a link to the specified message via DMs.
```

Reply:
https://github.com/python-discord/sir-lancebot/pull/929/commits/fbdfae42a40df0242d8050438d8c99aea193d386